### ISSUE TITLE
Unintended Behavior w/ str.strip() in genLinkerScript.py

### ISSUE THREAD

**Issue by 3096** [shibbo/OdysseyReversed#3]
https://github.com/shibbo/OdysseyReversed/blob/ddd7cd39e3833bfe5af2bb8848ba52a31fe593b1/scripts/genLinkerScript.py#L68
cuz it strips the leading "_" as well. Suggested fix:
```
			if symbolName.endswith("_0"):
				symbolName = symbolName[:-2]
```

**Comment by shibbo** [comment]
This makes no sense, why would I leave the _ there? No reason to keep it, everything links properly with both removed.

**Comment by 3096** [comment]
You have to keep the `_` in the front as it is part of the mangle. Without it, it will not "link properly" as the compiler will mangle it with `_` causing a mismatch.

**Comment by shibbo** [comment]
That’s not how str.strip() works. It will only remove instances of “_0”, not “_” and “0” seperately.

**Comment by shibbo** [comment]
Fixed with commit a9060c6.
